roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -0,0 +1,74 @@
+---
+
+- name: Install ManageIQ service account templates
+  copy:
+    src: manageiq-serviceaccount
+    dest: /etc/origin/
+
+- name: Add Managment Infrastructure project
+  command: > 
+    {{ openshift.common.admin_binary }} new-project
+    management-infra
+    --description="Management Infrastructure"
+  register: osmiq_create_mi_project
+  failed_when: "'already exists' not in osmiq_create_mi_project.stderr and osmiq_create_mi_project.rc != 0"
+  changed_when: false
```

Comment:
In the past, new-project will switch the currently configured project in the client config...  because of this, I would suggest copying the {{ openshift.common.config_base }}/master/admin.kubeconfig file to a temporary location and using that for creating the new project and the subsequent commands. You'll want to clean up the temporary file afterwards as well.